### PR TITLE
Propus-16.4.0-Zoom-get-token-script

### PR DESCRIPTION
**Pull request by ZurKaltura**
This change is [Reviewable](https://reviewable.io/reviews/kaltura/server/9471)

### CODE REVIEW

File: alpha/scripts/utils/getZoomToken.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?php
+
+if($argc != 2)
+{
+	echo "Arguments missing.\n\n";
+	echo "Usage: php getZoomToken.php {vendor account id}\n";
```
Comment (MosheMaorKaltura) [inline alpha/scripts/utils/getZoomToken.php:6]: instead of writing the script name - use argv[0] so we can rename it if needed.

Comment (MosheMaorKaltura) [inline alpha/scripts/utils/getZoomToken.php:6]: When you want to specify fields in the usage you should use < description > and not { }

File: alpha/scripts/utils/getZoomToken.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	exit;
+}
+
+$vendorAccountId = $argv[1];
+
+require_once(__DIR__ . '/../bootstrap.php');
+
+$zoomIntegration = ZoomHelper::getZoomIntegrationByAccountId($vendorAccountId);
```
Comment (MosheMaorKaltura) [inline alpha/scripts/utils/getZoomToken.php:14]: If not object return - let's print appropriate message and stop the scipt.

Comment (ZurKaltura) [reply alpha/scripts/utils/getZoomToken.php:14]: VerifyZoomIntegration take care of that

**Comment by MosheMaorKaltura** [comment]
reviewed

**Pull request merged**